similarity_api/src/routes/create.py

```diff
@@ -1,6 +1,8 @@
 from flask import Flask, jsonify, request, send_from_directory
 from routes.middleware import api_key_header
-from sim.compute import compute_goal_similarities, find_similar_goals, calculate_string_similarity
+from sim.compute import (calculate_string_similarity,
+                         compute_goal_similarities, find_similar_goals)
```

Comment:
if its going to be multi-line, please make each import on their own line

Reply:
I've been using [isort](https://pypi.org/project/isort/) to handle import linting and sorting so that we don't have to think about these things. I added a config (a2a08fedc5775eb438f580666b9384cddfd213e7) for isort and used `force_grid_wrap` which implements the rule you are looking for.